tests/migration_test.go

```diff
@@ -2882,53 +2882,29 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				Expect(supportedFeatures).Should(Not(ContainElement(false)),
 					"copy features must be supported on node")
 			}
-			getOtherNodes := func(nodeList *k8sv1.NodeList, node *k8sv1.Node) (others []*k8sv1.Node) {
-				for _, curNode := range nodeList.Items {
-					if curNode.Name != node.Name {
-						others = append(others, &curNode)
-					}
-				}
-				return others
-			}
-			isHeterogeneousCluster := func() bool {
-				nodes := libnode.GetAllSchedulableNodes(virtClient)
-				for _, node := range nodes.Items {
-					hostModel := getNodeHostModel(&node)
-					otherNodes := getOtherNodes(nodes, &node)
-
-					foundSupportedNode := false
-					foundUnsupportedNode := false
-					for _, otherNode := range otherNodes {
-						if isModelSupportedOnNode(otherNode, hostModel) {
-							foundSupportedNode = true
-						} else {
-							foundUnsupportedNode = true
-						}
-
-						if foundSupportedNode && foundUnsupportedNode {
-							return true
-						}
-					}
-				}
-
-				return false
-			}
 
 			It("[test_id:6981]should migrate only to nodes supporting right cpu model", func() {
-				if !isHeterogeneousCluster() {
-					log.Log.Warning("all nodes have the same CPU model. Therefore the test is a happy-path since " +
-						"VMIs with default CPU can be migrated to every other node")
+				sourceNode, targetNode, err := tests.GetValidSourceNodeAndTargetNodeForHostModelMigration(virtClient)
+				if err != nil {
+					Skip(err.Error())
 				}
 
-				By("Creating a VMI with default CPU mode")
-				vmi := alpineVMIWithEvictionStrategy()
-
-				if cpu := vmi.Spec.Domain.CPU; cpu != nil && cpu.Model != v1.CPUModeHostModel {
-					log.Log.Warning("test is not expected to pass with CPU model other than host-model")
+				vmi := libvmi.NewAlpine(
```

Comment:
I see that you've replaced `alpineVMIWithEvictionStrategy` with `libvmi.NewAlpine` but you didn't set the eviction strategy as it was with `alpineVMIWithEvictionStrategy`.

Reply:
Thanks for reviewing!
Done.